src/sentry/static/sentry/app/components/pagination.tsx

```diff
@@ -65,7 +65,7 @@ class Pagination extends React.Component<Props> {
               <IconChevron
                 direction="left"
                 size="sm"
-                color={previousDisabled ? 'gray200' : 'gray500'}
+                color={previousDisabled ? 'gray200' : 'purple300'}
```

Comment:
Hmm, how does this look w/o a `color` prop at all? It should inherit from the Button styles, no?

Reply:
Oh good point - I'll try and remove it